data/json/furniture.json

```diff
@@ -3381,5 +3381,207 @@
       "ter_set": "t_floor",
       "items": [ { "item": "pillow", "count": 19 }, { "item": "blanket", "count": 3 } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_file_cabinet",
+    "name": "filing cabinet",
+    "description": "A set of drawers in a sturdy metal cabinet, used to hold files.  It can be locked to protect important information.",
+    "symbol": "}",
+    "color": "dark_gray",
+    "move_cost_mod": 2,
```

Comment:
It is passable?

Reply:
Looks like that's from the mod. Same with the comment below, looks like it needs updating.

Reply:
It's just as passable as the countertop, which seems about right. The problem here is that the graphic shows something as tall as a locker, while the flags indicate it's probably one of the waist-high two drawer kind.

Reply:
Then if it's passable, it shouldn't be passable as easy as through the grass.

Reply:
It's the same move cost mod as every other waist-high flat-topped object, including ovens, countertops, tables, sinks, bathtubs, and more. If there's a problem here it's more likely to be the grass.

Reply:
Okay, I got it. Seems like we're gonna need the global audit of move costs for furniture and terrain after the 0.D.

Reply:
I think what it is is that the terrain move cost and the furniture are added together, so furniture with a move mod of 2 is still higher than terrain with that, but I'm not sure.